### DISCUSSION TITLE
Hugo resources

### DISCUSSION THREAD

**Discussion by LisaFC** [google/docsy#587] [General]
So I was just pointing a community member to the Hugo Discourse (https://discourse.gohugo.io/) to see if anyone in the Hugo community could help with an issue and I realized that we don't actually point people to it anywhere in our docs. 

- Where would be a good place to put this info in https://docsy.dev? Maybe a new section on our Community page, or a more general page on "Where to get help"? (including here!)
- Any other good Hugo resources and tips?

**Comment by narrenfrei** [comment]
On one hand the Community page would be the right place. But as I read most of the time in the Documentation section, also there a "Getting help" page would be well placed.

**Reply by LisaFC** [reply]
That's a good idea!
